diff --git a/ORST/Assets/Oculus/Interaction/Runtime/Scripts/Unity/PhysicsGrabbable.cs b/ORST/Assets/Oculus/Interaction/Runtime/Scripts/Unity/PhysicsGrabbable.cs
index 983db96..11d90cc 100644
--- a/ORST/Assets/Oculus/Interaction/Runtime/Scripts/Unity/PhysicsGrabbable.cs
+++ b/ORST/Assets/Oculus/Interaction/Runtime/Scripts/Unity/PhysicsGrabbable.cs
@@ -37,6 +37,9 @@ public class PhysicsGrabbable : MonoBehaviour
         [Tooltip("If enabled, the object's mass will scale appropriately as the scale of the object changes.")]
         private bool _scaleMassWithSize = true;
 
+        [Tooltip("If enabled, the PhysicsGrabbable will save the object's isKinematic state at startup instead of when the object is grabbed.")]
+        [SerializeField] private bool m_SaveInitialStateOnly = false;
+
         private bool _savedIsKinematicState = false;
         private bool _isBeingTransformed = false;
         private Vector3 _initialScale;
@@ -59,6 +62,10 @@ protected virtual void Start()
             this.BeginStart(ref _started);
             Assert.IsNotNull(_grabbable);
             Assert.IsNotNull(_rigidbody);
+
+            if (m_SaveInitialStateOnly) {
+                CachePhysicsState();
+            }
             this.EndStart(ref _started);
         }
 
@@ -102,7 +109,9 @@ private void HandlePointerEventRaised(PointerEvent evt)
         private void DisablePhysics()
         {
             _isBeingTransformed = true;
-            CachePhysicsState();
+            if (!m_SaveInitialStateOnly) {
+                CachePhysicsState();
+            }
             _rigidbody.isKinematic = true;
         }
 
